views/layout.jade

```diff
@@ -1,11 +1,11 @@
-!!! 5
+doctype html
 html
   head
     title #{settings.sitename} - #{title}
     link(rel='stylesheet', href='/stylesheets/bootstrap.min.css')
-    style(type='text/css')
-      body{padding-top:60px;}
+    link(rel='stylesheet', href='/stylesheets/custom.css')
```

Comment:
The inline style of 1 line here was added Explicitly to prevent the browser having to make another HTTP request for the sake of 3 lines of CSS.